lib/gitscrub.js

```diff
@@ -2,6 +2,7 @@ var rest = require('restler')
 var prompt = require('prompt')
 var RSVP = require('rsvp')
 var Lazy = require('lazy.js')
+var reposToScrub = require('./repos_to_scrub')
```

Comment:
Technically this is inside the node_module package if gitscrub was installed that way, should we leave it like that? On the one hand it makes it safer to leave it in the package, on the other, you don't have easy access to it like all other proeject files